mleap-spark-testkit/src/main/scala/org/apache/spark/ml/parity/SparkParityBase.scala

```diff
@@ -166,7 +166,7 @@ abstract class SparkParityBase extends FunSpec with BeforeAndAfterAll {
     }
   }
 
-  var relTolEps: Double = 1E-30
+  var relTolEps: Double = 1E-7
```

Comment:
The change 1E-30 is merged from master code. It break Scalar transformer test.
@ancasarb 
1E-30 is too small do not fit our numerical error. We should also update it in master.

Reply:
Could you send a PR to master?

Reply:
Create PR in master.
https://github.com/combust/mleap/pull/687